pkg/odo/cli/application/application.go

```diff
@@ -384,3 +99,14 @@ func printDeleteAppInfo(client *occlient.Client, appName string, projectName str
 	}
 	return nil
 }
+
+func validateApp(client *occlient.Client, appName, project string) error {
```

Comment:
I would rather have this called `ensureAppExists`, but it's too minor :)

Reply:
Better name! Done.